corehq/apps/app_manager/models.py

```diff
@@ -5056,28 +5061,29 @@ def create_all_files(self, build_profile_id=None):
             '{}suite.xml'.format(prefix): self.create_suite(build_profile_id),
             '{}media_suite.xml'.format(prefix): self.create_media_suite(build_profile_id),
         }
-        if self.target_commcare_flavor != 'none':
-            files['{}profile-{}.xml'.format(prefix, self.target_commcare_flavor)] = self.create_profile(
+        # ToDo: should add check for FF?
```

Comment:
I don't know that it matters if you check the flag or not. This is for the case where someone created a build for a specific flavor but then later turned off the feature flag, right?

Reply:
Ya.
If the flag is revoked for a domain, it would still add the files if the property was set though this would be a very very rare case, so would not be worth adding the toggle check each time files are created?

Reply:
yeah, exactly